config/hardhat-config.ts

```diff
@@ -94,6 +94,9 @@ export const NETWORKS_RPC_URL: iParamsPerNetwork<string> = {
   [eEthereumNetwork.main]: `https://eth-mainnet.alchemyapi.io/v2/${getAlchemyKey(
     eEthereumNetwork.main
   )}`,
+  [eEthereumNetwork.mainnet]: `https://eth-mainnet.alchemyapi.io/v2/${getAlchemyKey(
```

Comment:
Not sure why this command was necessary, but the mainnet v3 commands would fail with a network error unless there was an entry with the key "mainnet"

Reply:
Figured it out, issue was with `FORK=mainnet` in package.json command